server/routes/warningDetails.ts

```diff
@@ -85,6 +85,7 @@ export default function warningDetailsRoutes(
     )
 
     const failuresRecorded = createSelectItemListFromEnforceableContacts(warningDetails.enforceableContacts)
+    const contactListDeeplink = `${process.env.DELIUS_HOST}:${process.env.DELIUS_PORT}/NDelius-war/delius/JSP/deeplink.xhtml?component=ContactList&CRN=${breachNotice.crn}`
```

Comment:
It would be better to add an item to `config.ts`, and read it from there. Then you can set a default value (e.g. `http://localhost:7001`) when running locally.